ui/prometheus-plugin/src/utils/utils.test.ts

```diff
@@ -179,4 +187,19 @@ describe('getFormattedPrometheusSeriesName', () => {
     };
     expect(getFormattedPrometheusSeriesName(query, metric, series_name_format)).toEqual(output);
   });
+
+  it('should show correct raw series name', () => {
+    const query = 'node_load15{instance=~"(demo.do.prometheus.io:9100)"';
+    const metric = {
+      __name__: 'node_memory_Buffers_bytes',
+      env: 'demo',
+      instance: 'demo.do.prometheus.io:9100',
+      job: 'node',
+    };
+    const output = {
```

Comment:
minorrr, could be worth throwing in `series_name_format: undefined`, just to clear that this is the test case where `series_name_format` is undefined

Or adding that to name of test ("when series_name_format is undefined")